src/organization/organization.selectors.js

```diff
@@ -0,0 +1,48 @@
+import { createSelector } from 'reselect';
+
+const getOrganizationFromStore = state => state.organization;
+
+export const getOrganization = createSelector(
+  getOrganizationFromStore,
+  organization => organization.organization || {}
```

Comment:
Hey dude! I usually use it to query the store and do some computations based on that data when needed. The big plus I see is that we can compose any selectors we want, for example combine one selector from organizations, one from repos and two from auth, then produce a new prop to pass to any component we want. I truly believe this is a good adition to the stack.

Here's a copy paste from reselect's site

```
Selectors can compute derived data, allowing Redux to store the minimal possible state.

Selectors are efficient. A selector is not recomputed unless one of its arguments change.

Selectors are composable. They can be used as input to other selectors.
```